devlib/trace/ftrace.py

```diff
@@ -190,7 +214,7 @@ def start(self):
             self.logger.debug('Trace CPUIdle states')
             self.target.cpuidle.perturb_cpus()
         # Enable kernel function profiling
-        if self.functions:
+        if self.functions and self.tracer is None:
```

Comment:
`and self.tracer != 'function_graph'`?

Reply:
We don't want to execute this block when `self.trace == "function"` as trace-cmd will handle it by itself.